Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -362,8 +362,7 @@ RewardPledgeViewModelOutputs {
     self.expandRewardDescription = self.expandDescriptionTappedProperty.signal
 
     self.shippingLocationsLabelText = reward
-      .map { $0.shipping.summary }
-      .skipNil()
+      .map { $0.shipping.summary ?? "" }
```

Comment:
The shipping summary bug: if shipping data exists but the summary is returned as `nil` from the server, we'd display the storyboard content.